config/locales/test.en.yml

```diff
@@ -0,0 +1,17 @@
+en:
```

Comment:
Don't know if this is the best place to put it, but as we raise all our missing translations, we have some that are only used in tests

Reply:
This feels like a sign we should maybe get rid of them...although actually these are coming from test form definitions, right? Probably not too bad, and can't really think of anywhere better for them to go.